gap/almostsimple.gi

```diff
@@ -470,6 +472,7 @@ end;
 #        elordersstart := [31], numberrandgens := 2, tries := 1,
 #        triesforgens := 100, orblenlimit := 32, issimple := true ) );
 
+# if hint in AlmostSimpleHints exits appropriate function is executed
```

Comment:
```suggestion
# if hint in AlmostSimpleHints exists, appropriate function is executed
```
Also, hints in that db certainly do exist :-). Perhaps this should be more like "if a hint matching WHATEVER??? exists then..." ?